docs/knowledge/infra/nextjs-docker-healthcheck-ipv6-binding.md

```
@@ -0,0 +1,333 @@
+
+---
+
+## 6. 실무에서 권장되는 Next.js Docker HEALTHCHECK 패턴 (2026 기준)
+
+### 6.1 최소 구성 (우리 케이스 — 가장 흔함)
+
+```dockerfile
+ENV HOSTNAME="0.0.0.0"
+ENV PORT=3000
+
+HEALTHCHECK --interval=15s --timeout=3s --start-period=10s --retries=3 \
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)